### PR TITLE
Add SR2023 Kit Revision blog post

### PR DESCRIPTION
**Pull request by sedders123**

### CODE REVIEW

File: _posts/2022-10-27-sr2023-kit-revision.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: SR2023 Kit Revision
+---
+
+The Student Robotics Kit has received an upgrade! Thanks to the generosity of the Institution of Engineering and Technology and the Institution of Mechanical Engineers, we are now using Raspberry Pi computers in our kits. We've also developed an add-on board for the Pi (known as the KCH), which provides power to the Pi and several LEDs to indicate a robot's status.
+
+## Raspberry Pis
+
+Previously our kits used an ODROID U3+ as their brain, which served us well for many years. However, difficulties in procurement and teams needing increasing computing power led us to look for alternatives. We settled on using Raspberry Pi 4Bs with 2GB of RAM. These are easier to purchase, offer excellent computing performance, have built in WiFi, and much more.
```
Comment (PeterJCLaw) [inline _posts/2022-10-27-sr2023-kit-revision.md:9]: > These are easier to purchase

Well, they were :joy:

**Comment by PeterJCLaw** [comment]
General thought: maybe cross link to the relevant docs? Not sure if omitting that was intentional?

File: _posts/2022-10-27-sr2023-kit-revision.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## The KCH
+
+With a new powerful computer at the heart of our kit, we have also designed and built a new add-on HAT called the KCH. Named in memory of a Student Robotics volunteer who sadly passed away in 2020, it serves as a reminder of his love of robotics, innovation, and bees.
+
+The KCH is responsible for powering the Pi and providing output to users regarding the robot's status. The board has LEDs showing the boot progress of the Pi, indicators for the current status of the code, and most excitingly three user-programmable RGB LEDs. These LEDs provide an easy to use way to output what your robot is doing at that moment, which can be an invaluable debugging aid.
```
Comment (WillB97) [inline _posts/2022-10-27-sr2023-kit-revision.md:15]: Just feels like it reads better
```suggestion
The KCH is responsible for powering the Pi and providing output to users regarding the robot's status. The board has LEDs showing the boot progress of the Pi, indicators for the current status of the code, and most excitingly three user-programmable RGB LEDs. These LEDs provide an easy way to output what your robot is doing at that moment, which can be an invaluable debugging aid.
```

**Review by WillB97 — APPROVED** [review]
Looks good.

**Review by antoinepetty — APPROVED** [review]

**Pull request merged**